Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/CoreLights/DirectionalLightComponentConfig.h

```diff
@@ -62,6 +62,9 @@ namespace AZ
             //! Far depth clips for shadows.
             float m_shadowFarClipDistance = 100.f;
 
+            //! Whether this light enables shadow
+            bool m_shadowEnabled = true;
```

Comment:
Again, I might be missing this part of the logic, but If all lights enable shadows by default, which one is the casting one?

Reply:
There is a bus `ShadowingDirectionalLightNotificationsBus` to tell other directional lights to turn off shadow if one turns on its shadow.

In the case if all lights enable shadows by default, the latest one would cast shadow, this behavior is the same as the current behavior, but in a predictable manner